crates/sui-move/src/lib.rs

```diff
@@ -45,9 +45,10 @@ pub struct Calib {
 
 pub fn execute_move_command(
     package_path: Option<PathBuf>,
-    build_config: BuildConfig,
+    mut build_config: BuildConfig,
     command: Command,
 ) -> anyhow::Result<()> {
+    build_config.default_flavor = Some(move_compiler::editions::Flavor::Sui);
```

Comment:
Should we check for this flag being set to anything else? A bit strange to silently override it otherwise